### ISSUE TITLE
Training performance estimation retrieve

### ISSUE THREAD

**Issue by iXanthos** [keras-team/autokeras#1173]
Hello,

I would like to know can I retrieve the training performance printed during the `automl.fit()` call.
I have set the metric to `AUC` and the objective to `val_AUC`. During the training, the print shows this in the last iteration 

`65/65 [==============================] - ETA: 0s - loss: 0.4182 - AUC: 0.855 - ETA: 0s - loss: 0.3399 - AUC: 0.896 - ETA: 0s - loss: 0.3424 - AUC: 0.900 - ETA: 0s - loss: 0.3329 - AUC: 0.902 - 0s 4ms/step - loss: 0.3288 - AUC: 0.9032 - val_loss: 0.3411 - val_AUC: 0.9213`

Which one is these training performances I should look for when searching for the training performance estimation? Also, how can I retrieve it via python?

**Comment by haifeng-jin** [comment]
I cannot tell which one is the actual performance. We may need to clean up the output of AutoKeras later.
I think you can pass a callback function to the fit function. In that callback, you can track these metrics.

Thanks.
